### PR TITLE
Issue/16/create remind password view

### PR DESCRIPTION
**Pull request by Bedpaw**
to check

### CODE REVIEW

File: src/components/organisms/form/remind-password.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+<script>
+import AButtonSubmit from 'atoms/button/submit'
+import AInput from 'atoms/input'
+import { api } from 'api/index'
+import { formMixin } from 'mixins/form'
+
+export default {
+  name: 'remind-password.vue',
+  mixins: [formMixin],
```
Comment (Drego31) [inline src/components/organisms/form/remind-password.vue]: It should be `mixins: [mixins.form],`

File: src/components/organisms/form/remind-password.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    </div>
+    <p v-else>Link do odzyskania hasła wysłano na podany email.</p>
+  </div>
+</template>
+
+<script>
+import AButtonSubmit from 'atoms/button/submit'
+import AInput from 'atoms/input'
+import { api } from 'api/index'
+import { formMixin } from 'mixins/form'
```
Comment (Drego31) [inline src/components/organisms/form/remind-password.vue]: It should be `import { mixins } from 'mixins/base'`

File: src/components/pages/remind-password.vue [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  <t-page>
+    <div class="a-title">Przypomnij hasło</div>
+    <o-remind-password/>
+    <a-button @click="$router.push('/')">Wróć do logowania</a-button>
+  </t-page>
+</template>
+
+<script>
+import TPage from 'templates/page'
+import AButton from 'atoms//button'
```
Comment (Drego31) [inline src/components/pages/remind-password.vue]: It should be `import AButton from 'atoms/button'`

File: src/components/organisms/form/remind-password.vue (hunk 1 above)
Comment (Bedpaw) [reply src/components/organisms/form/remind-password.vue]: replaced

File: src/components/organisms/form/remind-password.vue (hunk 2 above)
Comment (Bedpaw) [reply src/components/organisms/form/remind-password.vue]: replaced

File: src/components/pages/remind-password.vue (hunk 3 above)
Comment (Bedpaw) [reply src/components/pages/remind-password.vue]: corrected

**Pull request merged**